### ISSUE TITLE
Review - Chp 3 - Translation and Review Guidelines

### ISSUE THREAD

**Issue by steffilazerte** [ropensci-review-tools/translation_guide#55]
I'll be dropping my comments regarding my review (PR #52 ) as issues for each chapter. This is a check list so you can keep track of which comments are resolved. As always, feel free to ignore anything that doesn't suite (just check it off as 'resolved' in that case).

- [x] Where is the list of active languages? Will it include contact information for the project managers?
- [x] Are the configuration files for the dev guide part of the workflow to produce the guide (my edits assume as much) or part of the guide text itself?
- [x] Here or elsewhere in the guide it might be worth linking to some reference for how to get started with git and GitHub? (if they're not already there and I just missed/forgot)
- [x] In the Review 1 example, the branch is called `softwarereview_policies.es.Rmd-en`... should it not be `softwarereview_policies.es.Rmd-es-auto`? (or at the very least, not `en` but `es` at the end?
- [x] I'm not sure that the Reviews need to be framed in the context of the example files as much as they are, I think it could be simpler to just say "e.g., file_name" as needed (see what you think of my suggestions)
- [x] I suggest moving the objectives to an untitled list at the start of the Review sections. They're not super long and it's nice to have an overview before getting into the details.
- [x] I think there are some final steps missing from this guide
    - How is this final review completed? On a separate branch again? In the main repository?
    - What happens to the translation once it is complete? Does some one need to be told to add the links to it somewhere?
    - If this goes beyond the guide, perhaps have a sentence or two saying something like "Now the translation is ready and rOpenSci staff will take it from here to ensure it is included along side the original material."
